index.html

```diff
@@ -1,110 +1,16 @@
 ---
 layout: default
 ---   
-<div class="greenhead">Where can I see the RFP?</div>
+
+{% for faq in site.data.faq %}
```

Comment:
That's fine, but the changes here don't really address the issue at hand, which is that the site is really unflexible if you need to add additional pages that might need questions and answers.